src/lib/cosmoz-omnitable-dropdown.js

```diff
@@ -49,145 +67,28 @@ export const renderDropdown = ({
-				font-size: 16px;
-				font-weight: normal;
-				--divider-color: var(
-					--paper-input-container-color,
-					rgba(0, 0, 0, 0.42)
-				);
-				--focused-color: #3f51b5;
-				width: 100%;
-				outline: none;
-			}
-
-			cosmoz-dropdown::before {
-				content: '';
-				position: absolute;
-				left: 0;
-				bottom: 0;
-				height: 1px;
-				width: 100%;
-				background: var(--divider-color);
-				pointer-events: none;
-				transition:
-					background 0.2s,
-					height 0.2s;
-			}
-
-			cosmoz-dropdown::after {
-				content: '';
-				position: absolute;
-				left: 0;
-				bottom: 0;
-				height: 1px;
-				width: 100%;
-				background: var(--focused-color);
-				transform-origin: center;
-				transform: scaleX(0);
-				pointer-events: none;
-				transition:
-					background 0.2s,
-					transform 0.2s,
-					height 0.2s;
-			}
-
-			cosmoz-dropdown.focused::after {
-				transform: scaleX(1);
-				height: 2px;
-			}
-
-			cosmoz-dropdown::part(button) {
-				background: transparent;
-				border-radius: unset;
-				position: relative;
-				width: 100%;
-				height: 100%;
-				min-height: 32px;
-				display: flex;
-				flex-direction: column;
-				justify-content: center;
-			}
-
-			.dropdown-button {
-				font-size: 16px;
-				box-sizing: border-box;
-				cursor: pointer;
-				color: var(--dropdown-button-color, rgba(0, 0, 0, 0.54));
-				position: absolute;
-				top: 50%;
-				transform: translateY(-50%);
-				transition:
-					transform 0.2s,
-					font-size 0.2s,
-					top 0.2s,
-					color 0.2s;
-				white-space: nowrap;
-				overflow: hidden;
-				text-overflow: ellipsis;
-				width: 100%;
-				pointer-events: none;
-			}
-
-			.left .dropdown-button,
-			.left .filter-value {
-				text-align: left;
-			}
-
-			.right .dropdown-button,
-			.right .filter-value {
-				text-align: right;
-			}
-
-			.center .dropdown-button,
-			.center .filter-value {
-				text-align: center;
-			}
-
-			.filter-value {
-				font-size: 16px;
-				text-align: left;
-				padding-bottom: 6px;
-				box-sizing: border-box;
-				white-space: nowrap;
-				overflow: hidden;
-				text-overflow: ellipsis;
-				color: var(--dropdown-filter-color, rgba(0, 0, 0, 0.87));
-			}
-
-			cosmoz-dropdown.filtered .dropdown-button {
-				top: 0;
-				transform: translateY(-100%);
-				font-size: 12px;
-			}
-
-			.float {
-				display: block;
-				height: 16px;
-				width: 100%;
-				content: ' ';
-			}
 		</style>
 
-		<div class="float"></div>
-
 		<cosmoz-dropdown
 			@focus=${onOpenedChanged}
 			@focusout=${onOpenedChanged}
-			class=${classMap(classes)}
+			class=${`${classMap(classes)} dropdown`}
 			title=${tooltip || ''}
 		>
-			<div slot="button" class="dropdown-button">${title || tooltip}</div>
-
-			${filterText
-				? html`<div slot="button" class="filter-value">${filterText}</div>`
-				: nothing}
-
-			<div class="dropdown-content" @mousedown=${(e) => e.stopPropagation()}>
+			<cosmoz-omnitable-dropdown-input
+				input-focused=${headerFocused}
```

Comment:
You should use a boolean attribute.

```suggestion
				?input-focused=${ifDefined(headerFocused)}
```